FinalProject/server.js

```diff
@@ -176,6 +176,9 @@ app.post('/usersandfollowrequests', function (req, res) {
 app.post('/adduser', function (req, res) { 
 
     var newEntry = req.body;
+    if(req.body.private == 'heyo') {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/wilsonz177/image-sharing-website/pullRequest?prid=6160981)